roles/kubernetes/master/tasks/post-upgrade.yml

```diff
@@ -1,6 +1,29 @@
 ---
+- name: "Post-upgrade | stop kubelet on all masters"
+  service:
+    name: kubelet
+    state: stopped
+  delegate_to: "{{item}}"
+  with_items: "{{groups['kube-master']}}"
+  when: needs_etcd_migration|bool
+
+- name: "Post-upgrade | Pause for kubelet stop"
+  pause:
```

Comment:
This logic doesn't even work well right now. It will upgrade, but you can't delete daemonsets (which means it fails later). By default, we pin back to etcd2 on upgrade.